apps/web/src/pages/EmployeeProfile/components/WfaSection/WfaSection.tsx

```diff
@@ -0,0 +1,369 @@
+        className="font-bold"
+        toggle={
+          <ToggleForm.LabelledTrigger
+            sectionTitle={intl.formatMessage(messages.wfa)}
+          />
+        }
+      >
+        {intl.formatMessage(messages.wfa)}
+      </ToggleSection.Header>
+      <ToggleSection.Content>
+        <ToggleSection.InitialContent>
+          {isNull ? <ToggleForm.NullDisplay /> : <Display user={user} />}
+        </ToggleSection.InitialContent>
+        <ToggleSection.OpenContent>
+          <FormProvider {...methods}>
+            <form onSubmit={handleSubmit(handleSave)}>
+              <Heading level="h4" size="h5" className="mt-0 font-bold">
+                {intl.formatMessage({
+                  defaultMessage: "Workforce adjustment information",
+                  id: "XBWnI6",
+                  description:
+                    "Title for the information specifically related to workforce adjustment",
+                })}
+              </Heading>
+              <RadioGroup
+                idPrefix="wfaInterest"
+                name="wfaInterest"
+                legend={intl.formatMessage(messages.wfaSituation)}
+                rules={{ required: intl.formatMessage(errorMessages.required) }}
+                items={wfaInterests}
+              />
+              {interest && interest !== WfaInterest.NotApplicable && (
+                <>
+                  <Heading level="h4" size="h5" className="font-bold">
+                    {intl.formatMessage({
+                      defaultMessage: "Key details",
+                      id: "GPizcR",
+                      description:
+                        "Title for key details related to workforce adjustment",
+                    })}
+                  </Heading>
+                  <p className="my-6">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "To be able to support you in the best way possible, it’s crucial to verify that the information about your current position is up to date.",
+                      id: "e4uyTF",
+                      description:
+                        "Description for the key details of workforce adjustment",
+                    })}
+                  </p>
+                  <Heading level="h5" size="h6">
+                    {intl.formatMessage(messages.currentSubstantivePosition)}
+                  </Heading>
+                  <SubstantiveExperiences
+                    query={unpackMaybes(user.currentSubstantiveExperiences)}
+                  />
+                  <DateInput
+                    name="wfaDate"
+                    id="wfaDate"
+                    legend={intl.formatMessage(messages.expectedEndDate)}
+                    content={intl.formatMessage({
+                      defaultMessage:
+                        "If possible, verify your expected end date.",
+                      id: "qDzAG6",
+                      description: "Help text for the wfa expected end date",
```

Comment:
Should be hidden in case of voluntary departure, alongside functional community stuff

Figma

<img width="468" height="599" alt="image" src="https://github.com/user-attachments/assets/665885a7-91cc-4f94-9ea6-cc5dde45a1f2" />

Current

<img width="889" height="829" alt="image" src="https://github.com/user-attachments/assets/92b74213-25ad-4f01-9986-7c0ae3f0814c" />

Reply:
Is this correct @Jerryescandon ? I feel as though people voluntarily leaving should still have the opportunity to set a date. They may know when they plan to leave.

Reply:
Hidden in [hide an null date field for voluntary departure](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14672/commits/51638aa89b77d2ea36543b19f02f7548d84dda14)